docs/ManualDeploymentExtended.md

```diff
@@ -416,14 +451,13 @@ Create a bitcoin.conf file to suit your needs. An example file that is suitable
 ```
 
 ```
-mainnet=1
 server=1                              # need RPC for btcpay.
 rpcbind=127.0.0.1                     # loopback is default for 0.18.0 but no harm making sure.
 whitelist=127.0.0.1                   # for nbxplorer.
 rpcallowip=127.0.0.1/32               # loopback is default but again no harm.
-disablewallet=1                       # btcpay does not host a bitcoin wallet.
```

Comment:
Why remove that?

Reply:
Because we do not disable the wallet anymore, even on the docker deploy